### PR TITLE
config-enhancement: allow PERM metamodel describe in multi-line

### PR DESCRIPTION
**Pull request by Outman9946**

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -72,7 +72,23 @@ func TestGet(t *testing.T) {
 
 	config.Set("other::key1", "test key")
 
-	if v := config.String("multi::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+	if v := config.String("multi1::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
```
Comment (hsluoyz) [inline config/config_test.go]: ``other::key1`` should also be renamed to ``multi1::name``.

File: config/config_test.go [hunk 2]
```diff
@@ -72,7 +72,23 @@ func TestGet(t *testing.T) {
...
+	}
+
+	if v := config.String("multi2::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
+	}
+
+	if v := config.String("multi3::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
```
Comment (hsluoyz) [inline config/config_test.go]: What's the difference between test ``multi2::name`` and ``multi3::name``?

File: config/config_test.go [hunk 3]
```diff
@@ -72,7 +72,23 @@ func TestGet(t *testing.T) {
...
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "r.sub==p.sub&&r.obj==p.obj", v)
+	}
+
+	if v := config.String("multi4::name"); v != "" {
+		t.Errorf("Get failure: expected different value for other::key1 (expected: [%#v] got: [%#v])", "", v)
+	}
+
+	if v := config.String("multi5::name"); v != "r.sub==p.sub&&r.obj==p.obj" {
```
Comment (hsluoyz) [inline config/config_test.go]: What's the difference between test ``multi1::name`` and ``multi5::name``?

Comment (Outman9946) [reply config/config_test.go]: Multi5 contains an EOF while mutil1 not

**Comment by hsluoyz** [comment]
The PR looks good now. But it's still not tested against a complete example. Can you add a ``TestRBACModelInMultLines`` test case here: https://github.com/casbin/casbin/blob/master/enforcer_test.go#L200 to test your PR? 

You can copy the code from: https://github.com/casbin/casbin/blob/master/model_test.go#L123-L134

**Comment by Outman9946** [comment]
Of course, coming soon.

**Comment by hsluoyz** [comment]
lgtm

**Pull request merged**